### PR TITLE
Various fixes for RESTier GA

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

**Comment by lewischeng-ms** [comment]
@karataliu Please help review. Thanks.

**Comment by karataliu** [comment]
looks good

File: src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs [hunk 1]
```diff
@@ -193,7 +193,7 @@ private ModelProducer()
                         true,
                         out storeGeneratedPattern);
 
-                    if (storeGeneratedPattern != null)
+                    if (storeGeneratedPattern != null && (string)storeGeneratedPattern.Value == "Computed")
```
Comment (congysu) [inline src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs]: Is this for #116?  @lewischeng-ms @challenh Can we have a sync on this?  How to represent `StoreGeneratedPattern.Identity` with OData annotation?

Comment (lewischeng-ms) [inline src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs]: Actually according to OData [standard vocabularies](http://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/vocabularies/Org.OData.Core.V1.xml), we don't support any annotation regarding StoreGeneratedPattern.Identity but only Computed.

**Pull request merged**